Xamarin.Forms.Platform.GTK/Renderers/ImageRenderer.cs

```diff
@@ -207,4 +214,38 @@ public sealed class UriImageSourceHandler : IImageSourceHandler
 			return image;
 		}
 	}
+
+
+	public sealed class FontImageSourceHandler : IImageSourceHandler
+	{
+		public Task<Pixbuf> LoadImageAsync(ImageSource imageSource,
+			CancellationToken cancellationToken = new CancellationToken(), float scale = 1)
+		{
+			var fontImageSource = imageSource as FontImageSource;
+			var fontPathAndFamily = fontImageSource.FontFamily.Split('#');
+			var privateFontCollection = new PrivateFontCollection();
+			privateFontCollection.AddFontFile(fontPathAndFamily[0]);
```

Comment:
Without the appropriate format (path#fontName) we could have an exception when trying to access an item by an index that does not exist, could you protect this case?.